roles-and-responsibilities.md

```diff
@@ -14,6 +14,28 @@ Admins are voted in by members every 6 months. To be eligible to apply to become
 * Acting as the main voice of the EMC
 * Enforcing the [Code of Conduct](code-of-conduct.md)
 
+### Criteria
+Nominated admins are evaluated on a case-by-case basis, but they are generally expected to meet some criteria.
+
+Must meet:
+
+* must be able to uphold the interests of the EMC above professional or personal interests
+* have a good understanding of what EMC is working towards and why
+* able to commit time to work on EMC for a minimum of 6 months
+
```

Comment:
Use uppercase at the start of each bullet to match the other bullets we have.

On the 2nd and 3rd bullet, change `EMC` to `the EMC`

Reply:
Updated